unittest.cfg

```diff
@@ -1,13 +1,19 @@
 [unittest]
 verbose = 2
-plugins = flufl.testing.nose
+plugins =
+    nose2.plugins.prof
 
 [log-capture]
 always-on = False
+clear-handlers = True
+date-format = None
+; format = %%(name)s: %%(levelname)s: %%(message)s
+format = %%(asctime)s: %%(name)s: %%(levelname)s: %%(message)s
+filter = -nose
+log-level = DEBUG
```

Comment:
Actually I'm still failing in capturing the logs so far. But these lines shouldn't mess up with the rest of the tests.